app-commons-test/src/test/scala/org/bitcoins/commons/DLCStatusTest.scala

```diff
@@ -1,13 +1,14 @@
-package org.bitcoins.core.protocol.dlc
+package org.bitcoins.commons
```

Comment:
This has to be moved because of the use of picklers which are defined in `app-commons`